### ISSUE TITLE
Discrepancy between the colours used in the hydrogen configs and the official palette

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#1802]
A discussion was started in [PR #1783](https://github.com/GCTC-NTGC/gc-digital-talent/pull/1783) about the discrepancy between the colours in the hydrogen configs and the official palette.  Once we get some guidance on the correct colours to use we should probably update our hydrogen config files and ensure that we are using the correct colours in our components - especially Button, Pill, and Chip.

**Comment by tristan-orourke** [comment]
@JoshBeveridge

**Comment by petertgiles** [comment]
I noticed there's now a warning about lightgray[.1] that was added in the chip PR.  That opacity will probably need to be added to the hydrogen config as well.

**Comment by gggrant** [comment]
@JoshBeveridge does this still need attention?

**Comment by JoshBeveridge** [comment]
Sorry, yes. The new palettes Brooke put together can be found in the documentation Figma file here: https://www.figma.com/file/5w15OeN9YiOR4UX0XKW7so/Digital-Talent---Design-documentation?node-id=201%3A1384

They're broken out to meet Hydrogen's configuration options, and we should follow the pattern of "primary", "secondary", "tertiary" instead of using color names. That way, for the IAP palette, you can use primaryIAP for the key or similar. This approach allows us to pivot colors on a dime, as opposed to find-replacing "purple" if we change to blue.

If I stop procrastinating in my spare time, I can push out 2.0.0 too... which will solve all this `[dark]gray` vs. `darkgray` balogna, so perhaps we wait until I get that together?

**Comment by tristan-orourke** [comment]
Blocked on H2 2.0

**Comment by petertgiles** [comment]
This is a very old issue and many colours have come and gone since then.  Probably no longer relevant.
